src/Controller/contact.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use App\Model\ContactController;
+
+if (isset($_POST["message"])) {
```

Comment:
this should be inside the controller and not in an isolated file or else mvc logic is compromised